### PR TITLE
[ci] crow-translate: update to 2.9.1

### PR DESCRIPTION
**Pull request by WhiredPlanck**
Topic Description
-----------------

Update `crow-translate` to 2.9.1

Package(s) Affected
-------------------

`crow-translate`

Security Update?
----------------

No

Test Build(s) Done
------------------

**Primary Architectures**

- [x] AMD64 `amd64`   
- [x] AArch64 `arm64`
    

**Secondary Architectures**

Architectural progress for "secondary," or experimental ports does not impede on merging of this topic.

- [x] Loongson 3 `loongson3`
- [x] PowerPC 64-bit (Little Endian) `ppc64el`

Update(s) Uploaded to Stable
----------------------------

**Primary Architectures**

- [ ] AMD64 `amd64`   
- [ ] AArch64 `arm64`

**Secondary Architectures**

Architectural progress for "secondary," or experimental ports does not impede on merging of this topic.

- [ ] Loongson 3 `loongson3`
- [ ] PowerPC 64-bit (Little Endian) `ppc64el`

### CODE REVIEW

**Comment by WhiredPlanck** [comment]
LGTM on AMD64

图片

File: extra-i18n/crow-translate/autobuild/defines [hunk 1]
```diff
@@ -1,5 +1,5 @@
 PKGNAME=crow-translate
 PKGSEC=utils
-PKGDES="A simple and lightweight translator that allows to translate and say selected text using Google, Yandex and Bing translate API"
+PKGDES="A simple and lightweight translator that allows to translate and say selected text using multiple translate API"
```
Comment (MingcongBai) [inline extra-i18n/crow-translate/autobuild/defines]: Again, keep it neutral and descriptive.

"A multi-backend online translator with pronunciation support"

**Review by MingcongBai — CHANGES_REQUESTED** [review]

File: extra-i18n/crow-translate/autobuild/defines (hunk 1 above)
Comment (WhiredPlanck) [reply extra-i18n/crow-translate/autobuild/defines]: > Again, keep it neutral and descriptive.
> 
> "A multi-backend online translator with pronunciation support"

Sounds great, will polish soon.

**Comment by WhiredPlanck** [comment]
Test builds all done.

**Review by MingcongBai — APPROVED** [review]

**Pull request merged**